core/src/main/scala/org/apache/spark/shuffle/sort/SortShuffleManager.scala

```diff
@@ -254,10 +256,18 @@ private[spark] object SortShuffleManager extends Logging {
     val executorComponents = ShuffleDataIOUtils.loadShuffleDataIO(conf).executor()
     val extraConfigs = conf.getAllWithPrefix(ShuffleDataIOUtils.SHUFFLE_SPARK_CONF_PREFIX)
         .toMap
+    val pluginOutputTrackingEnabled = conf.get(config.SHUFFLE_IO_PLUGIN_OUTPUT_TRACKING_ENABLED)
+    val trackerWorker = SparkEnv.get.mapOutputTracker.asInstanceOf[MapOutputTrackerWorker]
```

Comment:
This line may throw Exception, when running Spark local mode, the mapOuputTracker here is a `MapOutputTrackerMaster`